PlayCover/ViewModel/StoreVM.swift

```diff
@@ -191,6 +207,7 @@ class StoreVM: ObservableObject, @unchecked Sendable {
 struct SourceJSON: Codable, Equatable, Hashable {
     let name: String
     let data: [SourceAppsData]
+    let sourceURL: String
```

Comment:
Instead of having `sourceURL` here, it could be replaced with `id` and that could be used to compare if the source is enabled or not instead of by url. This only really matters if people have duplicate source urls, but is a nice-to-have.